recipe/meta.yaml

```
@@ -1,41 +1,142 @@
-{% set version = "2.5.2" %}
+{% set name = "openjpeg" %}
+{% set number = 0 %}
+{% set version = "2.5.4" %}
+# get "major.minor" version string, e.g. "2.5"
+{% set short_version = version.split('.')[:2] | join('.') %}
 
 package:
-  name: openjpeg
+  name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  url: https://github.com/uclouvain/openjpeg/archive/v{{ version }}.tar.gz
-  sha256: 90e3896fed910c376aaf79cdd98bdfdaf98c6472efd8e1debf0a854938cbda6a
+  - url: https://github.com/uclouvain/openjpeg/archive/v{{ version }}.tar.gz
+    sha256: a695fbe19c0165f295a8531b1e4e855cd94d0875d2f88ec4b61080677e27188a
+  # Data required to run the tests
+  - git_url: https://github.com/uclouvain/openjpeg-data.git
+    git_rev: 39524bd3a601d90ed8e0177559400d23945f96a9
+    folder: opj_data
+  # Following upstream pattern on known test failures
+  # See upstream repo; tools/travis-ci/knownfailures-*.txt
+  - path: tools/conda/knownfailures-win-64.txt
+  - path: tools/conda/knownfailures-linux-64.txt
+  - path: tools/conda/knownfailures-linux-aarch64.txt
+  - path: tools/conda/knownfailures-osx-arm64.txt
 
 build:
-  number: 1
-  run_exports:
-    # good compatibility in 2.x series, check before new release
-    # http://www.openjpeg.org/abi-check/timeline/openjpeg/
-    - {{ pin_subpackage('openjpeg') }}
-  ignore_run_exports:
-    # Needed as a transitive build-time dependency to enable PNG support but
-    # is not actually required by the output binaries.
-    - zlib
+  number: {{ number }}
+  # TODO: Temporarily skip Windows builds due to a PBP error
+  skip: true  # [win]
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
     - cmake
-    - make   # [unix]
-    - ninja  # [win]
+    - make                    # [unix]
+    - ninja                   # [win]
```

```suggestion
    - ninja-base                   # [win]
```

ninja-base without Python.